tests/benchmarks/test_benchmark.py

```diff
@@ -67,6 +69,20 @@ def protostar_cairo_path_fixture(request) -> Path:
     return (Path(request.node.fspath).parent.parent.parent / "cairo").absolute()
 
 
+def get_test_project_compiler(
+    include_paths: Optional[List[str]] = None,
+) -> ProjectCompiler:
+    # FIXME(arcticae): Build a project compiler with fake protostar.toml in the future if those tests are needed
+    return None  # type: ignore
+    # return ProjectCompiler(
+    #     project_root_path=project_root_path,
+    #     project_cairo_path_builder=project_cairo_path_builder,
+    #     contracts_section_loader=ProtostarContractsSection.Loader(
+    #         protostar_toml_reader
+    #     ),
+    # )
```

Comment:
Maybe this code should be deleted. It can be always restored from the git history.

Reply:
done